EDU-19094 - Atualizar 'Faturas - Visão Geral' (#935)
vtexdocs/help-center-content: docs/pt/tutorials/informacoes-de-faturamento/visao-geral-informacoes-de-faturamento.md

---
title: 'Informações de faturamento - Visão geral'
id: 6UxfCl4fw4GmyQwoUuIcQs
status: PUBLISHED
createdAt: 2018-05-17T20:22:30.358Z

> updatedAt: 2026-07-23T19:09:59.877Z

Author: ```suggestion
updatedAt: 2026-08-11T00:00:00.000Z
```

order: 1
---

**Informações de faturamento** é a página onde você administra todos os aspectos financeiros da sua conta VTEX. 

> ℹ️ Somente usuários com o perfil de acesso [Finance](/pt/docs/tutorials/perfis-de-acesso-predefinidos#finance), [Finance Full Access](/pt/docs/tutorials/perfis-de-acesso-predefinidos#finance-full-access) ou com recursos de [Billing](/pt/docs/tutorials/perfis-de-acesso-predefinidos#billing) vinculados ao perfil podem acessar esta seção do Admin VTEX. Para saber como configurar esses acessos, veja o artigo [Criar um perfil de acesso financeiro](/pt/docs/tutorials/criar-um-perfil-de-acesso-financeiro) ou solicite ao proprietário da loja que ajuste as permissões.

> Para acessar essa página, clique no seu avatar — identificado pela inicial do seu e-mail — no canto superior direito e, em seguida, em **Informações de faturamento**. Depois, selecione a página que deseja acessar:

Reviewer: ```suggestion
Para acessar essa página, clique no seu avatar — identificado pela inicial do seu e-mail — no canto superior direito da tela. Em seguida, clique em **Informações de faturamento** e selecione a página que deseja acessar:
```

> Para acessar essa página, clique no seu avatar — identificado pela inicial do seu e-mail — no canto superior direito e, em seguida, em **Informações de faturamento**. Depois, selecione a página que deseja acessar:

Reviewer: ```suggestion
Para acessar essa página, clique no seu avatar — identificado pela inicial do seu e-mail — no canto superior direito da tela. Em seguida, clique em **Informações de faturamento** e selecione a página que deseja acessar:
```

- [Dados cadastrais](#dados-cadastrais)
- [Contatos de faturamento](contatos-de-faturamento)

> ℹ️ Somente usuários com o perfil de acesso [Finance](/pt/docs/tutorials/perfis-de-acesso-predefinidos#finance), [Finance Full Access](/pt/docs/tutorials/perfis-de-acesso-predefinidos#finance-full-access) ou com recursos de [Billing](/pt/docs/tutorials/perfis-de-acesso-predefinidos#billing) vinculados ao seu perfil de acesso terão acesso a essa seção do Admin VTEX.

![visao-geral-informacoes-de-faturamento](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/informacoes-de-faturamento/visao-geral-informacoes-de-faturamento-1.jpg)

Veja abaixo mais detalhes sobre as diferentes seções e páginas deste recurso.

## Contratos

Na página **Contratos**, você encontra os contratos relacionados à sua loja. Algumas lojas podem ter mais de um contrato por terem adquirido diferentes produtos ou serviços VTEX.

## Faturas

Na página **Faturas**, você encontra um extrato com todos os títulos de pagamento da sua loja, organizados por mês. Cada título traz informações sobre o valor, a data de vencimento e o tipo de contrato que está sendo cobrado.

> >ℹ️ A VTEX é um serviço pós-pago: a cobrança é gerada no mês seguinte ao da utilização do serviço.

Author: ```suggestion
>ℹ️ A VTEX adota o modelo pós-pago, no qual a cobrança ocorre no mês seguinte ao da utilização do serviço. 
```

Para consultar o detalhamento de cada fatura, consulte o artigo Como verificar o detalhamento das minhas faturas.

## Dados cadastrais

> A página **Dados cadastrais** reúne as informações que você inseriu ao cadastrar a sua loja. Nessa página, você pode alterar apenas o seu nome e endereço. Para saber como alterar outras informações, leia o artigo Alterar os dados cadastrais da minha loja. 

Author: ```suggestion
A página **Dados cadastrais** reúne as informações que você inseriu ao cadastrar a sua loja. Estão disponíveis para edição apenas o seu nome e o seu endereço.

Para alterar outras informações, leia o artigo [Como verificar o detalhamento das faturas](/pt/docs/tutorials/como-verificar-o-detalhamento-das-faturas).
```

> Para alterar outras informações, leia o artigo [Alterar os dados cadastrais da loja](/pt/docs/tutorials/alterar-os-dados-cadastrais-da-loja).

Reviewer: ```suggestion
A página **Dados cadastrais** reúne as informações que você inseriu ao cadastrar a sua loja. Estão disponíveis para edição apenas o seu nome e o seu endereço. Para alterar outras informações, leia o artigo [Alterar os dados cadastrais da loja](/pt/docs/tutorials/alterar-os-dados-cadastrais-da-loja).
```

![visao-geral-informacoes-de-faturamento](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/informacoes-de-faturamento/visao-geral-informacoes-de-faturamento-1.jpg)

Veja abaixo mais detalhes sobre as diferentes seções e páginas deste recurso.

## Contratos

Na página **Contratos**, você encontra os contratos relacionados à sua loja. Algumas lojas podem ter mais de um contrato por terem adquirido diferentes produtos ou serviços VTEX.

## Faturas

Na página **Faturas**, você encontra um extrato com todos os títulos de pagamento da sua loja, organizados por mês. Cada título traz informações sobre o valor, a data de vencimento e o tipo de contrato que está sendo cobrado.

>ℹ️ A VTEX adota o modelo pós-pago, no qual a cobrança ocorre no mês seguinte ao da utilização do serviço. 

Para consultar o detalhamento de cada fatura, consulte o artigo Como verificar o detalhamento das minhas faturas.

## Dados cadastrais

A página **Dados cadastrais** reúne as informações que você inseriu ao cadastrar a sua loja. Estão disponíveis para edição apenas o seu nome e o seu endereço.

> Para alterar outras informações, leia o artigo [Como verificar o detalhamento das faturas](/pt/docs/tutorials/como-verificar-o-detalhamento-das-faturas).

Author: ```suggestion
Para alterar outras informações, leia o artigo [Verificar o detalhamento das faturas](/pt/docs/tutorials/verificar-o-detalhamento-das-faturas).
```
